### PR TITLE
Added object representations

### PR DESCRIPTION
**Pull request by cdce8p**
For `Accessory`, `Service` and `Characteristic`  
Added python venv suff to `.gitignore`.

### CODE REVIEW

File: pyhap/accessory.py [hunk 1]
```diff
@@ -165,6 +165,12 @@ def __init__(self, display_name, aid=None, mac=None, pincode=None,
 
         self._set_services()
 
+    def __repr__(self):
+        """Return the representation of the accessory."""
+        services = [s.display_name for s in self.services]
+        return "<accessory display_name='{}' services={}>" \
```
Comment (ikalchev) [inline pyhap/accessory.py]: Consistency using ticks around properties. Also I would write a capital A (for Accessories)

File: pyhap/characteristic.py [hunk 2]
```diff
@@ -114,6 +114,11 @@ def __init__(self, display_name, type_id, properties, value=None, broker=None):
         self.setter_callback = None
         self.hap_template = self._create_hap_template()
 
+    def __repr__(self):
+        """Return the representation of the characteristic."""
+        return "<characteristic display_name='{}' value={} properties={}>" \
+            .format(self.display_name, self.display_name, self.properties)
```
Comment (ikalchev) [inline pyhap/characteristic.py]: The second item should be `self.value`. Again the ticks (I get that you want to handle spaces in the display name, but maybe they could also prop in the value or properties)

**Comment by ikalchev** [comment]
Okay, let's leave the ticks as it is. For the names - I would either keep them the same or use a capital. I think it is more intuitive to expect to see the class name of some sort in the repr.

**Comment by cdce8p** [comment]
Wouldn't matter to me. What do you like more?

**Comment by ikalchev** [comment]
Honestly, capitals (`Accessory`, etc.). But I don't mind the way they are now, so whatever.

**Comment by cdce8p** [comment]
I personally would use the representation for debugging, so it wouldn't matter either :smile: 

I changed `self.display_state -> self.value`.

**Pull request merged**